Framework/Core/src/CCDBHelpers.cxx

```diff
@@ -74,7 +74,7 @@ CCDBHelpers::ParserResult CCDBHelpers::parseRemappings(char const* str)
     switch (state) {
       case IN_BEGIN: {
         if (*str == 0) {
-          return {remappings, "Empty string provided"};
```

Comment:
@ktf this was triggering an exception which supposed to print details https://github.com/AliceO2Group/AliceO2/blob/592ed27ed7deedf5931bed170f5ac612720eec22/Framework/Core/src/CCDBHelpers.cxx#L145 but instead was producing 
```
[1526580:internal-dpl-ccdb-backend]: [14:16:32][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-ctf-reader-workflow, device shutting down.
[1526580:internal-dpl-ccdb-backend]:  Reason:
[1526580:internal-dpl-ccdb-backend]:  Backtrace follow: 
```

Reply:
Yes, of course an empty string for the remappings is not an error.